odkx-src/services-managing.rst

```diff
@@ -61,6 +61,23 @@ Before you are able to synchronize data or application files to a device, you wi
 
       If you are using SSL, be sure to specify :code:`https://...`
 
+    .. _services-login-with-qr:
+
+    Additionally the app also provides an easy and quick option to login with QR code
```

Comment:
"Additionally" and "also" are redundant. The app is confusing, please name the app, as in "ODK-X Services also provides a quick and easy option to login with a QR code" looks like "a" is missing and also need period at end of sentence.